saw-remote-api/python/saw/llvm.py

```diff
@@ -23,6 +22,23 @@ def to_json(self) -> Any:
         server will dispatch on to then interpret the rest of the JSON object.``"""
         pass
 
+    @overload
+    def __getitem__(self, key : int) -> 'ElemVal':
```

Comment:
It might be worth mentioning that there is support for indexing in the documentation for `elem` and `field`.